src/Eloquent/Builder.php

```diff
@@ -233,8 +234,8 @@ public function createOrFirst(array $attributes = [], array $values = []): Model
             try {
                 $document = $collection->findOneAndUpdate(
                     $attributes,
-                    // Before MongoDB 5.0, $setOnInsert requires a non-empty document.
-                    // This should not be an issue as $values includes the query filter.
+                    // Before MongoDB 5.0, $setOnInsert requires a non-empty document,
+                    // this should not be an issue as $values include the query filter.
```

Comment:
Grammar was correct as-is. `$values` is a singular variable, so the verb would be plural.